scapy/route.py

```diff
@@ -21,8 +22,12 @@
 
 class Route:
     def __init__(self):
-        self.resync()
-        self.cache = {}
+        if isinstance(conf.route, list):
```

Comment:
I do not agree with this change. If I create a new `Route()` object, I expect it to be created from the current OS routes, not from a cache.